eblocker-ui/src/locale/lang-settings-de.json

```diff
@@ -417,6 +417,10 @@
                     "LABEL_CLOAKING_OFF_LABEL" : "Nicht tarnen",
                     "TAB_LABEL" : "Anonymisierung"
                 },
+                "DEVICE" : {
+                    "TAB_LABEL" : "Geräte",
```

Comment:
This should be singular, i.e. "Gerät"

Reply:
I wonder if we should show the device's name in the top bar, between the "Back to overview" button and the navigation buttons (|< < > >|).

Otherwise, when the user has gone to another tab, e.g. "Users", it's not clear which device's settings are displayed.

Reply:
@eblosug thanks for picking that up, it must of slipped passed me. I will correct it.
As to your suggestion, I think it’s a good idea, it’ll help maintain an overview when reviewing the other device tabs. I’ll implement the device name in the top bar. Thanks for taking a look.

Reply:
After looking at the front end, I'm reluctant to place the device name in the top bar, in favour of a homogenous approach, similar to other views (e.g. Blocker > Domain Ad Blocker). What do you think?

![device_name](https://user-images.githubusercontent.com/74534018/105771572-0b3dd480-5f61-11eb-9567-012a34d67c5a.jpg)

Reply:
Yes, you're right, it's unusual with regard to the other views. But on the other hand, if you don't place the device name in the top bar, the user has no idea which device she's currently configuring - think of the case where she works with several tabs or gets distracted for a few minutes. And as we also don't have a similar case in the other views, I think your design is good enough.